### PR TITLE
Privacy Sql Tracking Detection Using Easylist Adservers

### PR DESCRIPTION
**Pull request by hadiamjad**
New queries added for this year's chapter
- [ ] easylist_tracker_detection.sql

New util added for this year's chapter
- [ ] populate_easylist_adserver.py

### CODE REVIEW

**Comment by max-ostapenko** [comment]
Before pushing the changes, you can lint the code in your branch:
```
python3 -m venv src/.venv
source src/.venv/bin/activate
pip install -r src/requirements.txt
sqlfluff fix sql/2024/privacy/
```

File: sql/2024/privacy/tracking-detection/easylist_adservers.csv
Comment (max-ostapenko) [inline sql/2024/privacy/tracking-detection/easylist_adservers.csv:1]: As you have provided the script, no need for the data snapshot in the repo.
Please remove.

**Review by max-ostapenko — CHANGES_REQUESTED** [review]

**Comment by max-ostapenko** [comment]
@hadiamjad FYI I pointed this PR base to the Privacy chapter PR, so that we can merge all chapter in one step eventually.
Looking forward to your linting adjustments to merge this.

**Comment by tunetheweb** [comment]
> @hadiamjad FYI I pointed this PR base to the Privacy chapter PR, so that we can merge all chapter in one step eventually. Looking forward to your linting adjustments to merge this.

@max-ostapenko you might wanna update the Privacy chapter PR to latest version of `main` to hide the irrelevant changes in this PR (and stop those annoying diff comments being added to this PR, which happens when it thinks you're changing certain things).

**Comment by max-ostapenko** [comment]
> @max-ostapenko you might wanna update the Privacy chapter PR to latest version of `main` to hide the irrelevant changes in this PR (and stop those annoying diff comments being added to this PR, which happens when it thinks you're changing certain things).

@tunetheweb would you suggest to do it with merge commit or rebase to make it clean?

**Comment by tunetheweb** [comment]
I think a merge commit on the `privacy-sql-2024` branch will be fine. I hate rebasing :-)

**Review by max-ostapenko — APPROVED** [review]

**Comment by max-ostapenko** [comment]
> I think a merge commit on the `privacy-sql-2024` branch will be fine. I hate rebasing :-)

I merged `main` into Privacy PR and then also here - the diff was only growing 🤔 
Seems I still need to learn about git behavior :)

**Pull request merged**